guides/v2.2/cloud/integrations/cloud-integrations.md

```diff
@@ -35,3 +37,128 @@ magento-cloud integration:list
 +----------+--------------+---------------------------------------------------------------------------+
 ```
 {: .no-copy}
+
+## Generic webhooks
+
+You can capture events and `POST` JSON messages that report activity to a _webhook_ URL. Add a webhook URL using the following syntax:
+
+```bash
+magento-cloud integration:add --type=webhook --url=https://hook-url.example.com
+```
+
+-  `type`—Specify the `webhook` integration type.
+-  `url`—Provide the webhook URL that can receive JSON messages.
+
+The sample response shows a series of prompts that provide an opportunity to customize the integration. Using the default (blank) response sends all events on all environments in a project.
+
+You can customize the integration to report specific [events](#events-to-report), such as pushing code to a branch. For example, you can specify the `environment.push` event sends a message when a user pushes code to a branch:
+
+```terminal
+Events to report (--events)
+A list of events to report, e.g. environment.push
+Default: *
+Enter comma-separated values (or leave this blank)
+>
+```
+{: .no-copy}
+
+You can filter states, which can be `pending`, `in_progress`, or `complete`:
```

Comment:
This is OK, but it seems a little unclear. Consider:

 filter event reporting by states, which can include ....  and `complete" (Since you can enter more than one value )